src/Codeception/Test/Test.php

```diff
@@ -97,23 +123,16 @@ final public function run(\PHPUnit\Framework\TestResult $result = null): \PHPUni
             try {
                 $this->test();
                 $status = self::STATUS_OK;
-            } catch (\PHPUnit\Framework\AssertionFailedError $e) {
+            } catch (AssertionFailedError $assertionFailedError) {
                 $status = self::STATUS_FAIL;
-            } catch (\PHPUnit\Framework\Exception $e) {
+            } catch (Exception $exception) {
                 $status = self::STATUS_ERROR;
-            } catch (\Throwable $e) {
-                $e     = new \PHPUnit\Framework\ExceptionWrapper($e);
-                $status = self::STATUS_ERROR;
-            } catch (\Exception $e) {
-                $e     = new \PHPUnit\Framework\ExceptionWrapper($e);
+            } catch (Throwable $throwable) {
+                $throwable = new ExceptionWrapper($throwable);
                 $status = self::STATUS_ERROR;
             }
 
-            if (null !== $timer) {
-                $time = $timer->stop()->asSeconds();
-            } else {
-                $time = Timer::stop();
-            }
+            $time = null !== $timer ? $timer->stop()->asSeconds() : Timer::stop();
```

Comment:
Timer code can be simplified if we aren't going to support PHPUnit older than 9.2
https://github.com/Codeception/Codeception/pull/5933/files